pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -452,6 +453,20 @@ func mutateContainerResource(c *corev1.Container, override *v1beta1.OverrideSpec
 				}
 				c.Resources.Limits[corev1.ResourceMemory] = override.MemoryLimit
 			}
+			if override.LogLevel >= 0 {
+				// replace/append args with the new log level
+				index := -1
+				for i, arg := range c.Args {
```

Comment:
Instead of this find/replace logic, I think it would be simpler to just remove the verbosity arguments from the manifests and declare it in code. We can assign a default value in code if the override is unset

Reply:
the logic has been replaced to use default log level value when it's unset.